src/Tempest/Support/src/StringHelper.php

```diff
@@ -410,7 +410,14 @@ public function matches(string $regex): bool
 
     public function dd(mixed ...$dd): void
     {
-        dd($this->string, ...$dd); // @phpstan-ignore-line
+        ld($this->string, ...$dd); // @phpstan-ignore-line
```

Comment:
I'm confused—doesn't `dd` use `ld` under the hood? 

https://github.com/tempestphp/tempest-framework/blob/f54a0e0f30fcc9a7ea7b041d4a38e20181161aa2/src/Tempest/Debug/src/functions.php#L31-L37

Reply:
In theory yes.
But check this discuss here https://github.com/tempestphp/tempest-framework/pull/557#issuecomment-2405043570
In fact the `if (! function_exists())` comes after composer has load the `symfony/var-dumper` and so it doesn't run actually

Reply:
Oh, right! Thank you

Reply:
You're welcome 😉